src/app/js/admin/subresource/DeleteSubresourceButton.spec.js

```diff
@@ -57,7 +57,7 @@ describe('<DeleteSubresourceButton />', () => {
         );
 
         const okButton = wrapper.findWhere(node => {
-            return node.type() === Button && node.text() === 'OK';
+            return node.type() === Button && node.text() === 'Accept';
```

Comment:
We sure about this? What about translations?

Reply:
So far I haven't seen any translations in the tests (maybe I missed them). It would be interesting to put them in place but not the subject of the ticket in my opinion